### PR TITLE
docs: add stub instruction to docs README section

### PR DESCRIPTION
**Pull request by VidarDev**
If you only want to make documentation, you don't necessarily pay attention to the development section. That's why I have adapted it. I refer to this question from myself https://github.com/nuxt/framework/issues/3550#issuecomment-1061534210

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ We are using [Docus](https://nuxtlabs.com/docus) for documentation (*It is plann
 
 We recommend to install the [Docus extension](https://marketplace.visualstudio.com/items?itemName=NuxtLabs.docus) for VS Code.
 
+- Run `yarn stub` once in the root directory
```
Comment (danielroe) [inline README.md:63]:
```suggestion
- Run `npx yarn stub` once in the root directory
```

Comment (VidarDev) [reply README.md:63]: upps forget the npx >.>

**Review by danielroe — APPROVED** [review]
Thanks!

**Comment by VidarDev** [comment]
> Thanks!

I am always happy to help

**Pull request merged**